ui/components/ui/settings-icons/contacts-icon/contacts-icon.component.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Please drop the .component.js from all new component files added in this PR. this pattern exists from a time where we routinely had containers in separate files.